charts/tfy-otel-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: tfy-otel-collector
-version: 0.3.0-rc.1
+version: 0.3.0-rc.2
```

Comment:
do not update the version here directly. Create a tag in otel-collector repo for that